cli/popper/commands/cmd_stages.py

```diff
@@ -0,0 +1,45 @@
+import click
+import os
+import popper.utils as pu
+
+from popper.cli import pass_context
+from os.path import isfile
```

Comment:
You are importing `os` priorly and then a separate method from the same module which has been previously imported. I don't know if it's a good practice.

Reply:
@ankan17 I followed the same way it was written in cmd_init.py.Even, i found it odd.I think we should wait for @ivotron to comment.

Reply:
right, we should just do `import os` and then `os.path.isfile` whenever we need to use that function